### DISCUSSION TITLE
How do i create account on noctua software?

### DISCUSSION THREAD

**Discussion by hemraj007** [geneontology/helpdesk#482] [Q&A]
I am a research scholar at SC&SS, JNU New Delhi in India.
I am currently working on biomedical text mining. For a project i need to utilize GO-CAM. Can someone help me with creating account to access noctua software?

**Comment by balhoff** [comment]
@hemraj007 I believe the Noctua accounts are solely used for data curation by the GO contributors. Do you just want to access all the existing GO-CAMs? They are all stored as Turtle files in this Git repository: https://github.com/geneontology/noctua-models

You can also download a Blazegraph journal containing all the models for a given release, along with some precomputed OWL inferences: http://release.geneontology.org/2024-04-18/products/blazegraph/index.html
